stdlib/builtins.pyi

```diff
@@ -89,7 +89,7 @@ class object:
     @property
     def __class__(self: _T) -> Type[_T]: ...
     @__class__.setter
-    def __class__(self, __type: Type[object]) -> None: ...  # noqa: F811
+    def __class__(self, __type: Type[object]) -> None: ...  # type: ignore # noqa: F811
```

Comment:
Would be nice to add a comment explaining what is being ignored here

Reply:
Sure, I can do that. There didn't seem to be a precedent for this, which is why I initially left it out.

Reply:
Thanks! I think it's helpful in the future for documenting why we have these type ignores.